src/components/sbb-expansion-panel-header/sbb-expansion-panel-header.scss

```diff
@@ -49,17 +49,20 @@
   display: flex;
 }
 
+.sbb-expansion-panel-header__icon {
+  width: var(--sbb-expansion-panel-icon-size);
```

Comment:
To achieve a consistent design, regardless what consumer slottes, I decided to fix the width and scale the icon